local-dev/rabbitmqqueues.bicep

```diff
@@ -17,17 +17,14 @@ limitations under the License.
 @description('Information about what resource is calling this Recipe. Generated by Radius. For more information visit https://docs.radapp.dev/operations/custom-recipes/')
 param context object
 
-@description('Name of the queue. Defaults to the name of the Radius RabbitMQ resource.')
```

Comment:
Why is this being removed? Usually we default these to be the name of the resource but allow them to be overridden